_data/projects/jayk-gupta-piano.yml

```
@@ -0,0 +1,10 @@
+name: Piano
+desc: Made a piano using Html, CSS and vanilla JavaScript,designed using vanilla CSS.You can play it both with mouse nad keyboard.
+site: https://github.com/jayk-gupta/Piano
+tags:
+  - html
+  - css
+  - javascript
+upforgrabs:
+  name: up for grabs
```

:art: this is what the `name` here should be

```suggestion
  name: good first issue
```